styles/styles.css

```diff
@@ -0,0 +1,93 @@
+
+*{
+
+  padding: 0px;
+  border: 0;
+  margin: 0;
+  height: auto;
+  font-family: 'Roboto', sans-serif;
+
+}
+body
+{
```

Comment:
Put this '{' on the same line with the selector, like this `body {`. Make this change everywhere please